### PR TITLE
Added Like, Comments, bookmarks and follow functionality

### PR DESCRIPTION
**Pull request by vrinda-mahajan**
src/components/sidebar/rightSidebar.jsx
src/features/bookmark/bookmark.jsx
src/features/post/comment.jsx
src/features/post/post.jsx

Review only the changes of this [Commit](https://github.com/vrinda-mahajan/social-media-react/pull/3/commits/a2bbd1ec96fe88d8418bd5d09a421feae61c5530) only.
Thanks for reviewing.

### CODE REVIEW

File: src/features/bookmark/bookmark.jsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import { Box, Flex, Heading, useColorModeValue } from "@chakra-ui/react"
+import { LeftSidebar, RightSidebar } from "components"
+import { Post } from "features"
+
+const { useSelector } = require("react-redux")
```
Comment (devsurajrai) [inline src/features/bookmark/bookmark.jsx]: You can also directly import useSelector from 'react-redux'

**Review by devsurajrai — APPROVED** [review]
Great Job . Everything looks great. You can improve component's naming a bit. Keep it up.

**Review by hsnice16 — APPROVED** [review]
Great work. Nothing new to suggest, just do the changes that other reviewer has suggested. Good Job.

**Comment by vrinda-mahajan** [comment]
Thanks, @devsurajrai, and @hsnice16  for reviewing.

**Pull request merged**